tex/paper.tex

```diff
@@ -125,15 +125,15 @@ \section*{Keywords}
 \section*{Introduction}
 
 Several Iterated Prisoner's Dilemma tournaments have generated much interest;
-including Axelrod's original tournaments \cite{Axelrod1980a,Axelrod1980b},
-two 2004 anniversary tournaments \cite{kendall2007iterated}, and
-the Stewart and Plotkin 2012 tournament \cite{Stewart2012}, following the
-discovery of zero-determinant strategies.  Subsequent research has spawned an
-enormous number of papers, but rarely are the results reproducible. Among
-well-known tournaments, in only one case is the full original source code
+Axelrod's original tournaments \cite{Axelrod1980a,Axelrod1980b}, two 2004
+anniversary tournaments \cite{kendall2007iterated}, and the Stewart and Plotkin
+2012 tournament \cite{Stewart2012}, following the discovery of zero-determinant
+strategies.  Subsequent research has spawned a number of papers (many of which
+are referenced throughout this paper), but rarely are the results reproducible.
+Amongst well-known tournaments, in only one case is the full original source code
```

Comment:
This was specifically requested by one of the reviewers.  I personally don't care but I suggest we go with their preference?